infrastructure/README.md

```diff
@@ -0,0 +1,61 @@
+# AIDA Interchange Format Infrastructure
```

Comment:
I think this could be either AIDA Validation Infrastructure, or AIF Validation Infrastructure.  But the important thing is that it's a validation infrastructure.

Reply:
Yea i'll change it to AIDA Validation Infrastructure.